web/server/codechecker_server/api/report_server.py

```diff
@@ -1998,37 +1994,20 @@ def getCheckerDoc(self, checkerId):
          - checkerId
         """
 
-        missing_doc = "No documentation found for checker: " + checkerId + \
-                      "\n\nPlease refer to the documentation at the "
+        return ""
 
-        if "." in checkerId:
-            sa_link = "http://clang-analyzer.llvm.org/available_checks.html"
-            missing_doc += "[ClangSA](" + sa_link + ")"
-        elif "-" in checkerId:
-            tidy_link = "http://clang.llvm.org/extra/clang-tidy/checks/" + \
-                checkerId + ".html"
-            missing_doc += "[ClangTidy](" + tidy_link + ")"
-        missing_doc += " homepage."
+    @exc_to_thrift_reqfail
+    @timeit
+    def getCheckerLabels(self, checkers):
+        result = []
 
-        try:
-            md_file = self.__checker_doc_map.get(checkerId)
-            if md_file:
-                md_file = os.path.join(self.__checker_md_docs, md_file)
-                try:
-                    with open(md_file, 'r',
-                              encoding='utf-8',
-                              errors='ignore') as md_content:
-                        missing_doc = md_content.read()
-                except (IOError, OSError):
-                    LOG.warning("Failed to read checker documentation: %s",
-                                md_file)
-
-            return missing_doc
+        for checker in checkers:
+            result.append(list(map(
+                lambda x: f'{x[0]}:{x[1]}',
+                self._context.checker_labels.labels_of_checker(
+                    checker.checkerId, checker.analyzerName))))
```

Comment:
We can simplify the logic this way:
```suggestion
        return [
            list(map(lambda x: f'{x[0]}:{x[1]}',
                self._context.checker_labels.labels_of_checker(
                    checker.checkerId, checker.analyzerName)))
            for checker in checkers]
```